packages/compiler-dom/__tests__/transforms/vModel.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { parse, transform, generate, CompilerOptions } from '@vue/compiler-core'
+import { transformModel } from '../../src/transforms/vModel'
+import { DOMErrorCodes } from '../../src/errors'
+import { transformElement } from '../../../compiler-core/src/transforms/transformElement'
+import { transformExpression } from '../../../compiler-core/src/transforms/transformExpression'
+function parseWithVModel(template: string, options: CompilerOptions = {}) {
+  const ast = parse(template)
+
+  transform(ast, {
+    nodeTransforms: [transformExpression, transformElement],
+    directiveTransforms: {
+      ...options.directiveTransforms,
+      model: transformModel
+    },
+    ...options
+  })
+
+  return ast
+}
+
+describe('compiler: transform v-model', () => {
+  test('simple expression', () => {
+    const root = parseWithVModel('<input v-model="model" />')
```

Comment:
We need to assert that `root.helpers` contains the correct directive symbol for each element/input type supported.

Reply:
I already change my code.